src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -168,8 +171,9 @@ def __init__(self, cython_call: cygrpc._AioCall, metadata: MetadataType,
         self._response_deserializer = response_deserializer
 
     def __del__(self) -> None:
-        if not self._cython_call.done():
-            self._cancel(_GC_CANCELLATION_DETAILS)
+        if hasattr(self, '_cython_call'):
```

Comment:
I'm wondering when it can happen if `_cython_call` is an attribute initialized at construction time.

Reply:
Under unclear condition either the `_cython_call` is deallocated prior to `Call` or the initialization failed, I observed an exception during development that said attribute `_cython_call` not found while invoking `__del__`. It is safer to add this guarding statement.

Reply:
weird ... I would put a comment at least.

Reply:
Adding comment `# The '_cython_call' object might be destructed before Call object`